zowe-install/src/main/resources/component-scripts/start.sh

```diff
@@ -112,14 +112,14 @@ _BPX_JOBNAME=${ZOWE_PREFIX}${GATEWAY_CODE} java -Xms32m -Xmx256m -Xquickstart \
     -Dapiml.service.hostname=${ZOWE_EXPLORER_HOST} \
     -Dapiml.service.port=${GATEWAY_PORT} \
     -Dapiml.service.discoveryServiceUrls=https://${ZOWE_EXPLORER_HOST}:${DISCOVERY_PORT}/eureka/ \
-    -Dapiml.service.preferIpAddress=true \
-    -Dapiml.service.allowEncodedSlashes=true \
-    -Dapiml.service.corsEnabled=false \
+    -Dapiml.service.preferIpAddress=${APIML_PREFER_IP_ADDRESS} \
```

Comment:
Why only one service?

Reply:
I will change also others, the PR is not ready as I need basically to use this specific version of API ML in the zowe-install-packaging, so I needed to run the pipeline with the "Publish PR artifact" option enabled.

Reply:
i see, thank you